hypershift-operator/controllers/supportedversion/reconciler.go

```diff
@@ -81,7 +83,8 @@ func (r *Reconciler) ensureSupportedVersionConfigMap(ctx context.Context) error
 		if cm.Data == nil {
 			cm.Data = map[string]string{}
 		}
-		cm.Data[configMapKey] = string(contentBytes)
+		cm.Data[ConfigMapVersionsKey] = string(contentBytes)
+		cm.Data[ConfigMapServerVersionKey] = version.String()
```

Comment:
one more nit: should this just be `version.GetRevision()` ?

Reply:
I don't know, the parallelism seems more apt for users - why would we prefer this? I know that the whole range is reported in the other key, but I can't see why giving the full version is undesirable